crud/crud.py

```diff
@@ -152,7 +152,8 @@ async def git_verify(self, ctx, filepath):
                                       email or "famiel@tsubakibot.com")
             commiter = pygit2.Signature("Famiel", "famiel@tsubakibot.com")
             parent, ref = repo.resolve_refish(refish=repo.head.name)
-            repo.create_commit(ref.name, author, commiter, "Updating "+filepath, tree, [parent.oid])
+            filename_idx = filepath.rfind("/")
+            repo.create_commit(ref.name, author, commiter, f"Updating {filepath if filename_idx == -1 else filepath[filename_idx+1: ]}", tree, [parent.oid])
```

Comment:
Are you looking for [`os.path.basename`](https://docs.python.org/3/library/os.path.html#os.path.basename)? Using `rfind` that way would fail on Windows, which usually uses backslash as the path separator ([`os.sep`](https://docs.python.org/3/library/os.html#os.sep), but you shouldn't check for it since Windows can use either).

By the way, the splitting idea is better expressed as:
```py
filename = filepath.rsplit("/", maxsplit=1)[-1]
```
which splits from the end, but at most once, and then returns the last part. (It wastes time creating the first part, but a file path is short enough that it's negligible here.)